libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.spec.ts

```diff
@@ -645,9 +645,9 @@ describe('Autocomplete component', () => {
 
       const dropdownElement = getSearchResultsContainer();
       const autocompleteElement = getAutocompleteElement();
-      const formattedWidth = `${
-        autocompleteElement.getBoundingClientRect().width
-      }px`;
+      const formattedWidth = `${+autocompleteElement
```

Comment:
I was seeing issues here where the precision of this number was causing issues - it may have been a machine thing. I was getting this width as `1019.2000122070312px` but the test was expecting `1019.2px`. I just updated these tests to give a precision of 2 decimals and then the `+` removes any trailing zeros.

Reply:
ooh I didn't know `+` did that, cool